README.rst

```diff
@@ -117,6 +117,15 @@ The following is the output from ``ansible-lint --help``, providing an overview
                             ".ansible-lint"
       --version             show program's version number and exit
 
+CI/CD
+-----
+
+If execution under `Github Actions`_ is detected via the presence of
+``GITHUB_WORKFLOW`` variable, the linter will also print errors using their
+`annotation`_ format.
+
+.. _GitHub Actions: https://github.com/features/actions
+.. _annotation: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-error-message
```

Comment:
Here's how you can match the length of the paragraph above
```suggestion
.. _annotation:
   https://docs.github.com/en/actions/reference
   /workflow-commands-for-github-actions#setting-an-error-message
```

Reply:
If our linters jobs is not complaining lets not go into this. URLs are known to cause problems
when wrapped and I do not want to risk.